sktime/forecasting/var.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+__all__ = ["VectorAutoRegression"]
+__author__ = ["Taiwo Owoseni"]
+
+from statsmodels.tsa.api import VAR as _VAR
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+import pandas as pd
+
+# data type is multivariate
+
+
+def is_Dataframe(Z):
+
+    if isinstance(Z, pd.DataFrame) and len(Z.columns) > 1:
+        Z = Z.select_dtypes(include=["float", "int"])
+        return Z
+    else:
+        raise ValueError("Accepts only DataFrame. But found `type(Z)` ")
+
+
+def to_dataframe(Z):
+
+    pass
+
+
+class VectorAutoRegression(_StatsModelsAdapter):
+    # _fitted_param_names = ("aic", "bic", "aicc", "hqic")
+    _fitted_param_names = ("aic", "fpe", "hqic", "bic")
+    # _required_parameters = ["Z"]
+    _tags = {
+        "univariate-only": False,
+        "requires-fh-in-fit": False,
+        "handles-missing-data": False,
+    }
+
+    def __init__(self, Z):
```

Comment:
data should not be passed to the constructor. Constructor arguments should be hyper-parameters or components, in this case it should be the hyper-parameters you can choose in `statsmodels` `VAR`.

Reply:
Thank you Franz . I will made necessary adjustments